app/locustio/jira/http_actions.py

```diff
@@ -118,10 +122,11 @@ def create_issue_submit_form():
         r = locust.client.post('/secure/QuickCreateIssue.jspa?decorator=none', params=issue_body,
                                headers=ADMIN_HEADERS, catch_response=True)
         content = r.content.decode('utf-8')
-
+        if not ('"id":"project","label":"Project"') in content:
+            logger.error(params.err_message_create_issue)
```

Comment:
need to add content like:
logger.error(f'User {user[0]} authentication failed: {content}')